lib/hanami/lambda/rack.rb

```diff
@@ -41,19 +41,7 @@ def call(event:, context:)
       #
       # @since 0.1.0
       def build_env(event, headers, context)
-        {
-          ::Rack::REQUEST_METHOD => event["httpMethod"],
-          ::Rack::PATH_INFO => event["path"] || "",
-          ::Rack::RACK_VERSION => ::Rack.release,
-          ::Rack::RACK_INPUT => StringIO.new(event["body"] || ""),
-          ::Hanami::Lambda::LAMBDA_EVENT => event,
-          ::Hanami::Lambda::LAMBDA_CONTEXT => context
-        }.tap do |env|
-          content_type = headers.delete("Content-Type") ||
```

Comment:
To get "content-type" with different keys because all values are possibly received from Lambda event. We should not change it.